charms/feast-ui/terraform/outputs.tf

```
@@ -0,0 +1,12 @@
+output "app_name" {
+  value = juju_application.feast_ui.name
+}
+
+
+output "requires" {
+  value = {
+    feast-configuration = "feast-configuration",
```

convention for terraform is underscore (see https://developer.hashicorp.com/terraform/language/style#code-style)
```suggestion
    feast_configuration = "feast-configuration",
```